### ISSUE TITLE
Breaking change with createGlobalStyle in 1.7.0

### ISSUE THREAD

**Issue by MatthewLarner** [zacanger/styled-reset#7]
# Is this a feature request or a bug? Bug

# Expected behavior:  Expect this package to still work with styled-components 3 as it's listed in the peer deps
```
  "peerDependencies": {
    "styled-components": ">=3.0.0 || >=4.0.0"
  },
```

# Actual behavior: It doesn't work with styled-components 3 as it's using `createGlobalStyle` which was introduced with styled-components 4

# Environment info (relevant browser, OS, etc.; run `npx envinfo`):

I'm using styled-components@3.4.10

I guess probably should either be a breaking change and remove styled-components 3 from peer dependencies as unsupported or revert and patch(or both).

**Comment by zautumnz** [comment]
It's probably [this line](https://github.com/zacanger/styled-reset/blob/v1.7.0/src/index.js#L60) causing things to blow up with styled-components 3.x. I probably should've actually tested, but figured it would be safe as long as someone on 3.x didn't try to import/require the named export `Reset`.

I think what I'll do is revert that in a patch, but also publish a new major version of styled-reset, and recommend in the README that sc 3.x users use the previous tag. Thanks for the bug report!

**Comment by zautumnz** [comment]
I just published a patch at `styled-reset@1.7.1`. Sorry about the bug, and thanks for the issue!

**Comment by MatthewLarner** [comment]
Cheers mate, much appreciated.
Thanks for the fast turn around!
